Documentation updated (#113)
It4innovations/hyperqueue: CHANGELOG.md

# v0.4.0

## New features

  * Streaming - streaming stdout/stderr of all tasks in a job into one file
    to avoid creating many files.
  * Setting a priority via ``hq submit --priority <P>``

>   * Option ``hq submit --wait ...`` to wait until the submit is not finished

Reviewer: ```suggestion
  * Option ``hq submit --wait ...`` to wait until the submitted job finishes
```

  * Command ``hq wait <id> / all / last`` to wait for a given job(s)
  * Setting a priority via ``hq submit --priority <P>``

>   * Command ``hq resubmit <job-id>`` to resubmit a previous task

Reviewer: The command mentions `job-id`, but the text mentions task (ID)
